### PR TITLE
one-liner changes to remove examples/*.rs compilation error

### PR DESCRIPTION
**Pull request by tjyang**
Hi There
I went through all example/*.rs and found these small one-liner changes to  remove compilation  errors. Tested against rust 1.64.

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
Nice! Please fix the CI errors and I'll merge this then!

File: examples/tcp.rs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+extern crate rexpect;
```
Comment (petreeftime) [inline examples/tcp.rs:1]: This is not required anymore since edition 2018.

**Comment by petreeftime** [comment]
I'm not entirely sure what this fixes: the error module is called `error` and not `errors`. https://github.com/rust-cli/rexpect/blob/master/src/lib.rs#L69

**Comment by tjyang** [comment]
@matthiasbeyer 
This PR should be closed. lets do one at a time.

**Comment by tjyang** [comment]
https://github.com/rust-cli/rexpect/issues/81 created.

**Pull request closed without merging**